index.js

```diff
@@ -1,25 +1,6 @@
 /*! MIT License © Sindre Sorhus */
 
-const globals = {};
-
-const globalProperties = [
-	'Headers',
-	'Request',
-	'Response',
-	'ReadableStream',
-	'fetch',
-	'AbortController',
-	'FormData'
-];
-
-for (const property of globalProperties) {
-	Object.defineProperty(globals, property, {
-		get() {
-			const value = globalThis[property];
-			return typeof value === 'function' ? value.bind(globalThis) : value;
-		}
-	});
-}
+const globals = globalThis;
```

Comment:
Can we drop this too and just use `globalThis` directly everywhere?

Reply:
Sure, done. I find it a bit lengthy and strange to look at. Maybe I just need to get used to it.